### PR TITLE
po: Update Portuguese translation

### PR DESCRIPTION
**Pull request by hugok79**

### CODE REVIEW

File: po/pt.po [hunk 1]
```diff
@@ -167,89 +206,150 @@ msgstr ""
...
 
-#: src/exm-comment-dialog.blp:35 src/exm-detail-view.blp:35
-#: src/exm-detail-view.blp:168 src/exm-detail-view.c:350
+#. Translators:
+#. - '%s' is an extension e.g. Blur my Shell
+#. - Please use unicode quotation marks e.g. “” (not "")
+#: src/exm-browse-page.c:292
+#, c-format
+msgid "e.g. “%s”"
+msgstr "por exemplo, \"%s\""
```
Comment (mjakeman) [inline po/pt.po:240]:
```suggestion
msgstr "por exemplo, “%s”"
```

Could we use unicode quotation marks (“”) here?

**Review by mjakeman — COMMENTED** [review]
Hi, everything looks good to me.

One comment about punctuation. Could we use unicode quotation marks in the line above? (Does it make sense for Poruguese to use these?)

Thanks for the contribution!

File: po/pt.po (hunk 1 above)
Comment (hugok79) [reply po/pt.po]: Fixed

**Comment by mjakeman** [comment]
Great, thanks for the contribution!

**Pull request merged**